TAO/TAO_IDL/fe/idl.ypp

```diff
@@ -2365,10 +2369,8 @@ annotation_dcl
         : IDL_ANNOTATION_DECL defining_id '{'
         {
           if (idl_global->idl_version_ < IDL_VERSION_4)
-            {
-              idl_global->err ()->idl_version_error (
-                "Annotations are an IDL4 feature");
-            }
+            idl_global->err ()->idl_version_error (
+              "Annotations are not allowed in IDL3");
```

Comment:
I would prefer to refer to IDL4, in the past we had IDL1/2/3, maybe some people are using IDL2, at that moment IDL3 will maybe confuse them (also for the other errors about IDL3)

Reply:
I'm trying to make the messages future proof, assuming there's an IDL 5 and if for some reason we haven't ditched TAO_IDL, we've updated it. Also the `idl_version` isn't aware of IDL versions before 3, so I don't consider them to be relevant.

Reply:
The user may not be aware of that.  So this could be "...not allowed in IDL versions before 4" or something like that.